static/js/reducers/dashboard.js

```diff
@@ -34,7 +35,11 @@ export const dashboard = (state: DashboardsState = INITIAL_DASHBOARDS_STATE, act
   case REQUEST_DASHBOARD: // eslint-disable-line no-case-declarations
     let newBaseState = R.dissoc(username, state);
     if (action.payload === true) {
-      return updateStateByUsername(newBaseState, username, INITIAL_DASHBOARD_STATE);
+      return updateStateByUsername(
+        newBaseState,
+        username,
+        _.merge({}, INITIAL_DASHBOARD_STATE, { noSpinner: true })
```

Comment:
We need to also set `fetchStatus` here. Instead of having the `if (action.payload === true)` we could just have

    {
      fetchStatus: FETCH_PROCESSING,
      noSpinner: action.payload,
    }